cdec/installation-manager-resources/src/main/resources/patches/5.0.0-M4/single_server/patch_before_update.sh

```diff
@@ -71,3 +71,10 @@ updateMigrationConfig admin_ldap_mail "${OLD_admin_ldap_mail}"
 
 # https://github.com/codenvy/deployment/tree/master/automation/jpa-migration-tool
 ${CODENVY_IM_BASE}/jre/bin/java -jar ${CURRENT_DIR}/jpa-migration-tool.jar migrate -config-file $MIGRATION_CONF &>> $LOG_FILE
+
+# create update info file
+echo >> $LOG_FILE
+echo "Create update info file" >> $LOG_FILE
+echo "All users' passwords have been reset. This is a one-time event to increase security with an internal database consolidation.
+Admin user name: '${OLD_admin_ldap_user_name}'
+Admin password:  '${OLD_admin_ldap_password}'" > ${PATH_TO_UPDATE_INFO}
```

Comment:
I see you print several lines into the file. So use just several ```echo``` commands. It will be more obvious.